.github/workflows/publish.yml

```diff
@@ -30,6 +30,11 @@ jobs:
       - name: 'Build project using dotnet'
         run: dotnet build ${{ env.SOLUTION_PATH }} --no-restore --configuration ${{ env.BUILD_CONFIGURATION }} -p:ContinuousIntegrationBuild=true
 
+      - name: Reduce AspectInjector requirement to 2.8.1
```

Comment:
Add comment here, for future updates please
This workaround should be documented as well